stylesheetFinal.css

```diff
@@ -9,97 +9,81 @@ body,
 main {
     height: 100%;
     margin: 0;
-    /* height: 100%;
-    padding: 0;
-    border: 0; */
 }
 
 @font-face {
     font-family: Explogos;
     src: url(Explogos\ -\ Demo.ttf);
 }
 
-.outside-flex-wrapper {
-    margin-top: 40px;
-    margin: 40px auto 0;
+.outside-wrapper {
+    height: 100%;
+    max-width: 1000px;
+    margin: 0 auto;
+    padding: 60px 80px;
+}
+
+.flex-wrapper {
     display: flex;
-    max-width: 1050px;
-    gap: calc(40px + 2vw);
-    padding: 0px 20px 0px 20px;    
+    gap: 80px;
+    padding: 0px 20px;    
 }
 
 .left {
-    /* width: 420px; */
-    /* width: 30.7vw; */
-    width: calc(300px + 8.7vw);
+    width: 100%;
 }
 
 header {
-    /* position: relative;
-    top: 0; */
-    /* width: 33vw; */
-    /* margin: 70px auto 30px 140px; */
+    display: block;
     margin: 0 auto 30px -50px;
 }
 
 h1 {
     color: rgb(173, 18, 102);
     font-family: 'Explogos', 'Gill Sans', 'Gill Sans MT', sans-serif;
-    /* font-size: 2.3vw; */
     font-weight: 500;
     letter-spacing: 8px;
-    /* letter-spacing: 0.59vw; */
     line-height: 1.3em;
-    /* line-height: 3.3vw; */
     text-transform: uppercase;   
 }
 
 hr {
-    margin: -10px 0 40px 0;
-    
     height: 0.3px;
-    /* height: 0.04vh; */
     width: 45px;
-    /* width: 8%; */
+
+    margin: -10px 0 40px 0;
     
     background-color:rgb(173, 18, 102);   
 }
 
 textarea {
     height: 250px;
-    /* height: 32.6vh; */
     width: 100%;
         
     background-color: white;
     border: none;
     outline: none;
-    /* padding: 0; */
+    padding: 0;
     
     font-size: 1em;
     overflow: hidden;
 }
```

Comment:
Is there a pattern for these empty lines? I suggest avoiding leaving empty lines in css blocks. Maby you can leave an empty line, if there are a lot of properties in one css block, but with up to 10 there is no need to.